translations/en-us.json

```diff
@@ -328,9 +328,7 @@
       "support": "Support",
       "reportissue": "Report Issue",
       "theme": "Theme <a href='https://github.com/almasaeed2010/AdminLTE'>AdminLTE</a> made with ♥ by <a href='https://almsaeedstudio.com/'>Almsaeed</a>",
-      "createIssue": "Create Issue",
-      "createProject": "Create Project",
-      "createConversation": "Create Conversation",
+      "createModule": "Create Module",
```

Comment:
Too technical, either call it 'New' or 'Create' or 'Create New' or 'Create Record'. I think 'Create' or 'New' will be better since they are small and good enough.

While researching for reference, I came about another message and that is '+' button, no.
JIRA, Create : https://community.atlassian.com/forums/image/serverpage/image-id/72166i659DC4F29880DC84?v=v2
Asana, + : https://assets.asana.biz/transform/323dd590-8ca8-4713-a6b7-cc5b7b88a76f/updated-team-page-marketing-team-1
Gitlab uses a +
Github uses a +

Reply:
Implemented "+"